sql/set.go

```diff
@@ -94,7 +94,7 @@ func (p *planner) SetTimeZone(n *parser.SetTimeZone) (planNode, *roachpb.Error)
 			location = "UTC"
 		}
 		if _, err := time.LoadLocation(location); err != nil {
-			return nil, roachpb.NewError(err)
+			return nil, roachpb.NewUErrorf("cannot find %s: %v", location, err)
```

Comment:
If `err` is known to be non-nil, `%s` and `%v` are equivalent. If `err` may be nil, `%v` is a little nicer (https://play.golang.org/p/6g1NUFTI3r). I prefer `%v` for this reason but `%s` is currently more common in cockroach code.